{"name": "Логин с неверным логином/паролем (параметризация)", "status": "passed", "steps": [{"name": "Авторизируемся в приложении", "status": "passed", "start": 1750510292300, "stop": 1750510292548}], "parameters": [{"name": "wrong_logout_data", "value": "{'email': 'ktrof_006@yandex.ru', 'password': 'wrong_ktrof_006@yandex.ru'}"}], "start": 1750510292300, "stop": 1750510292548, "uuid": "d2c0aa84-6b54-42a5-a5c8-c3ed28213035", "historyId": "1a8158e5dbbb18c24565e67ed54ea5aa", "testCaseId": "04493e2aa7ea31843048e03f3ff14117", "fullName": "tests.test_login.TestLogin#test_login_with_wrong_data_status_code_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Rumata"}, {"name": "thread", "value": "11016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}